api/src/pcapi/routes/serialization/offerers_serialize.py

```diff
@@ -268,15 +269,17 @@ class CreateOffererQueryModel(BaseModel):
     _validate_phone_number = validation.phone_number_validator("phoneNumber", nullable=True)
 
 
+# bulle
 class SaveNewOnboardingDataQueryModel(BaseModel):
+    activity: offerers_models.OnboardingActivity | None
     address: offerers_schemas.AddressBodyModel
     createVenueWithoutSiret: bool = False
     isOpenToPublic: bool
     publicName: str | None
     siret: str
     target: Target
     token: str
-    venueTypeCode: str
+    venueTypeCode: str | None
```

Comment:
Je pense que l'on pourrait aussi mettre ici:

```suggestion
    venueTypeCode: offerers_models.VenueTypeCode | None
```